test/unit/cog_api/fake/bundles_test.exs

```diff
@@ -79,15 +78,15 @@ defmodule CogApi.Fake.BundlesTest do
     end
 
     it "will allow a string for enabled" do
-      bundle = %Bundle{name: "a bundle", enabled: true}
-      bundle = Client.bundle_create(fake_endpoint, bundle) |> get_value
+      config = minimal_config
+      bundle = Client.bundle_create(fake_endpoint, config) |> get_value
 
       {:ok, updated_bundle} = Client.bundle_update(
         fake_endpoint,
         bundle.id,
-        %{enabled: "false"}
+        %{enabled: "true"}
```

Comment:
There isn't currently a way to create a bundle that is already enabled. You have to create it and then enable it. Since this test was just checking to see if you could set the enabled flag with a string, I just flipped it instead of doing two updates.